### PR TITLE
Make some rules descriptions generic and add ubuntu specific bits

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- A few rules mentioned specific distros, but such rules can also be used by other distros. So replaced the distro's name with {{{ full_name }}}.
- Also added Ubuntu checks and specific texts.

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]

**Review by cipherboy — APPROVED** [review]
Just commenting that I've reviewed this internally and merged this stuff there; I'd prefer someone from Red Hat to do final review and merge though :-)

File: linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml [hunk 1]
```diff
@@ -58,6 +68,8 @@ ocil: |-
     <pre>$ grep -i upgradeRemoveDroppedPackages {{{ pkg_manager_config_file }}}</pre>
     The output should return something similar to:
     <pre>solver.upgradeRemoveDroppedPackages=true</pre>
+    {{% elif 'ubuntu' in product %}}
+    <pre>$ grep -i remove-unused {{{ pkg_manager_config_file }}}</pre>
```
Comment (yuumasato) [inline linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml:72]: This is will render the `ocil` for Ubuntu as just the grep command. Is that intentional?

Comment (dodys) [reply linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml:72]: Nice catch! 
I just pushed a fix for that.

Comment (yuumasato) [reply linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml:72]: The first sentence will still be the `grep` command, :grin: 

Just wanted to point out that the other `ocil` start with:
`To verify that <tt>...</tt> is configured properly, run the
    following command:`

**Review by yuumasato — APPROVED** [review]
LGTM.

**Comment by ggbecker** [comment]
@cipherboy We are rewriting some parts of the `display_login_attempts` in #6990 and I noticed you guys added `ubuntu` in the rule description but I think you also should edit the OVAL, ansible, bash... if it configures in `/etc/pam.d/login`

Can you please check this?

**Comment by cipherboy** [comment]
@ggbecker Not sure I follow. :-) Right now we haven't upstreamed any of our non-templated hardening changes -- mostly they reside in `ubuntu.sh` files (including for `display_login_attempts`) and thus aren't impacted by #6990 merging. #6990 already merged, so I'm guessing it is a little late for us to review it, but are you asking us to fix anything wrong (if anything is)?

Note that while we use the same path as SUSE (which seems to have regressed in #6990), we have different arguments (we still allow silent for instance).

HTH

**Pull request merged**